client/core/core.go

```diff
@@ -1173,7 +1574,7 @@ func (c *Core) handleReconnect(uri string) {
 }
 
 // handleOrderBookMsg is called when an orderbook response is received.
-func (c *Core) handleOrderBookMsg(dc *dexConnection, msg *msgjson.Message) error {
+func handleOrderBookMsg(c *Core, dc *dexConnection, msg *msgjson.Message) error {
```

Comment:
Since these don't actually use `Core`, can we make `c` into `_` so it's clearly intended that way.